.github/workflows/publish.yml

```diff
@@ -0,0 +1,33 @@
+name: Publish Package
+on:
+  release:
+    types: [created]
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      packages: write
+    strategy:
+      matrix:
+        include:
+          - registry-url: 'https://registry.npmjs.org'
+            command: npm publish --access public
+            token: NPM_TOKEN
```

Comment:
Is this a secret? Should it be `${{ secrets.NPM_TOKEN }}`?

Reply:
It's actually not possible to pass in secretes into the matrix. It's a workaround, to access the secret later down the road.